apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -1,7 +1,6 @@
 ---
```

Comment:
The snapshot change is due to my re-ordering the enum variants of `RouterSelector`.